packages/admin/admin/src/mui/menu/Item.tsx

```diff
@@ -118,14 +28,15 @@ const Item: React.FC<WithStyles<typeof styles> & MenuItemProps & MuiListItemProp
 }) => {
     const context = React.useContext(MenuContext);
     if (!context) throw new Error("Could not find context for menu");
-    if (level > 2) throw new Error("Maximum nesting level of 2 exceeded.");
+    if (level > 3) throw new Error("Maximum nesting level of 2 exceeded.");
```

Comment:
```suggestion
    if (level > 3) throw new Error("Maximum nesting level of 3 exceeded.");
```

Reply:
2 times is correct. level 1 is root and you can nest two times ;)